pkgs/tools/misc/fontforge/default.nix

```diff
@@ -9,22 +9,30 @@
 
 stdenv.mkDerivation rec {
   name = "fontforge-${version}";
-  version = "20160404";
+  version = "20170730";
+
+  # The way $version propagates to $version of .pe-scripts (https://github.com/dejavu-fonts/dejavu-fonts/blob/358190f/scripts/generate.pe#L19)
+  SOURCE_DATE_EPOCH = lib.fileContents (runCommand "unixtime-of-${version}" {} "date -d ${version} +%s > $out");
```

Comment:
Doesn't look like this is used on the Nix side, so I'd expect you could do this within the builder to avoid the need for IFD. Is that not the case?